test/DynamoCoreWpfTests/SerializationTests.cs

```diff
@@ -855,7 +902,9 @@ public void NotesSerializeAsAnnotations()
         public object[] FindWorkspaces()
         {
             var di = new DirectoryInfo(TestDirectory);
-            var fis = di.GetFiles("*.dyn", SearchOption.AllDirectories);
+            //var fis = di.GetFiles("*.dyn", SearchOption.AllDirectories);
+            var fis = new string[] { "*.dyn", "*.dyf" }
+            .SelectMany(i => di.GetFiles(i, SearchOption.AllDirectories));
             return fis.Select(fi => fi.FullName).ToArray();
         }
```

Comment:
I am currently running all the serialization tests to make sure this works as expected